wordpress/wp-content/plugins/cds-base/classes/Modules/FormRequestSite/Setup.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CDS\Modules\FormRequestSite;
+
+use CDS\Modules\FormRequestSite\Block;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\RequestException;
+use CDS\Modules\Notify\NotifyClient;
+
+class Setup
+{
+    public function __construct()
+    {
+        add_action('wp_enqueue_scripts', [$this, 'enqueue']);
+
+        /*
+         * Note - if testing with WP ENV
+         * https://wordpress.org/support/topic/wp-env-with-gutenber-doesnt-have-a-rest-api/
+         */
+        add_action('rest_api_init', function () {
+            register_rest_route('request/v1', '/process', [
+                'methods' => 'POST',
+                'callback' => [$this, 'confirmSend'],
+                'permission_callback' => function () {
+                    return '';
+                }
+            ]);
+        });
+
+        new RequestSite();
+        new Block();
+    }
+
+    public function enqueue()
+    {
+        wp_enqueue_script('cds-request-js', plugin_dir_url(__FILE__) . '/src/handler.js', ['jquery'], "1.0.0", true);
+
+        // where is this?
```

Comment:
Believe we can remove this --

CDS_VARS is localized in another script ... this creates an object on the Window (window.CDS_VARs)